application/views/surveyAdministration/tabCopy_view.php

```diff
@@ -7,7 +7,7 @@
     <div class="grow-10 ls-space padding left-10 right-10">
         <!-- copy survey form -->
         <?php echo CHtml::form(array('surveyAdministration/copy'), 'post', array('id'=>'copysurveyform', 'name'=>'copysurveyform', 'class'=>'form30 row')); ?>
-            <div class="col-4">
+            <div class="col-md-4">
```

Comment:
I would just do "col-12 col-md-4"... adding col-12 just in case.

Reply:
> I would just do "col-12 col-md-4"... adding col-12 just in case.

I check : really not needed :)